apps/web/screens/Organization/Fiskistofa/ShipDetails/ShipDetails.tsx

```diff
@@ -0,0 +1,199 @@
+
+import * as styles from './ShipDetails.css'
+
+const TAB_OPTIONS = ['aflamarks', 'deilistofna'] as const
+
+interface ShipDetailsProps {
+  locale: Locale
+  namespace: Record<string, string>
+  organizationPage: Query['getOrganizationPage']
+  ship: Query['getSingleShip']
+}
+
+const ShipDetails: Screen<ShipDetailsProps> = ({
+  locale,
+  namespace,
+  organizationPage,
+  ship,
+}) => {
+  const router = useRouter()
+
+  const n = useNamespace(namespace)
+
+  const breadcrumbItems = [
+    {
+      title: 'Ísland.is',
+      href: linkResolver('homepage', [], locale).href,
+    },
+    {
+      title: 'Fiskistofa',
+      href: linkResolver('organizationpage', ['fiskistofa'], locale).href,
+    },
+  ]
+
+  const { shipNumber, selectedTab } = useMemo(() => {
+    let shipNumber = null
+    const nr = router?.query?.nr
+    if (nr && typeof nr === 'string' && !isNaN(Number(nr))) {
+      shipNumber = Number(nr)
+    }
+    const value = TAB_OPTIONS.find((o) => o === router?.query?.tab)
+    const selectedTab = value ?? TAB_OPTIONS[0]
+    return { shipNumber, selectedTab }
+  }, [router?.query])
+
+  return (
+    <>
+      <OrganizationHeader organizationPage={organizationPage} />
+      <GridContainer>
+        <Box marginTop={6} marginBottom={6}>
+          <Breadcrumbs items={breadcrumbItems} />
+        </Box>
+
+        <Box marginBottom={4}>
+          <Stack space={1}>
+            <Text variant="h1">{ship.name}</Text>
+            <Box className={styles.shipNumber}>
+              <Text fontWeight="semiBold" color="white">
+                {shipNumber}
+              </Text>
+            </Box>
+          </Stack>
+        </Box>
+
+        <Box className={styles.searchBox} marginBottom={4}>
+          <SidebarShipSearchInput {...namespace} label="" />
+        </Box>
+
+        <Box>
+          <Inline alignY="center" space={5}>
+            {TAB_OPTIONS.map((tab) => {
+              const isSelected = selectedTab === tab
+              return (
+                <Link
+                  shallow={true}
+                  href={{
+                    pathname: router.pathname,
+                    query: {
+                      nr: shipNumber,
+                      tab: tab,
+                    },
+                  }}
+                  underline="normal"
+                  key={tab}
+                  underlineVisibility={isSelected ? 'always' : undefined}
+                  color={isSelected ? 'blue400' : undefined}
+                >
+                  {n(tab, `Reiknivél ${tab}`)}
+                </Link>
+              )
+            })}
+          </Inline>
+        </Box>
+      </GridContainer>
+
+      <Box className={styles.container}>
+        <GridContainer>
+          {selectedTab === 'aflamarks' && shipNumber && (
+            <AflamarkCalculator shipNumber={shipNumber} namespace={namespace} />
+          )}
+          {selectedTab === 'deilistofna' && shipNumber && (
+            <DeilistofnaCalculator
```

Comment:
Just fish food for thought :fish: :fish: 
Question regarding how much we want to keep our code conventions for english.

Aflamark is "Catch quota" in english.
Deilistofn, when related to fish, is "Straddling stock".

Reply:
Aah ok, thanks for that. I'll change the names :D